### PR TITLE
Features/activity class

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
## Is this a fix or a feature?
This is a feature.

## What is the change?
Adds an activity class to the activity.js file. Also applies minor refactoring to CSS styling.

## What does it fix?
Spacing errors in CSS have been removed.

## Where should the reviewer start?
Check activity.js to ensure class was created properly with framework in place for all necessary attributes and methods. 
Check styles.css for spacing errors within code.

## How should this be tested?
This does not add functionality so nothing can be tested.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1 +1,161 @@
...
+  border-radius: 5px;
+  padding-bottom: 7px;
+}
+
+.time-box {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+
```
Comment (Shakikka) [inline styles.css:98]: Extra space

Comment (Shakikka) [inline styles.css]: Space delete

File: activity.js [hunk 2]
```diff
@@ -1 +1,25 @@
...
+
+  markComplete() {
+    this.completed = true;
+  }
+
+  saveToStorage() {
+
+  }
+
+}
```
Comment (Shakikka) [inline activity.js:23]: ; after }

**Review by Shakikka — APPROVED** [review]
This class outline looks peachy, and besides an extra return in the css file, and a missing semicolon on line 23 of activity.js, everything else looks stellar.

File: styles.css (hunk 1 above)
Comment (BigBike96) [reply styles.css]: Good catch on the spaces

**Review by BigBike96 — APPROVED** [review]

**Pull request merged**